examples/ads.amp.html

```diff
@@ -1414,5 +1415,21 @@ <h2 id="yandex">Yandex</h2>
       type="zucks"
       data-frame-id="_bda46cf5ac">
   </amp-ad>
+
+  <amp-analytics>
```

Comment:
Please do not change add example to the ads example page. As we talked offline, if you want you can add something to `fakead3p.amp.html`, and `_ping_.js` However I think it makes sense to merge it with the existing page that tests user error.